web-services-app/src/test/java/edu/unc/lib/boxc/web/services/utils/DownloadTestHelper.java

```diff
@@ -15,5 +18,7 @@ public static void assertCorrectImageReturned(byte[] actualContent) throws IOExc
         assertArrayEquals(imageContent, actualContent);
     }
 
-
+    public static PID makePid() {
```

Comment:
There is already TestHelper.makePid available in the service app, might as well reuse it